frontend/src/components/note/NoteCard.vue

```diff
@@ -0,0 +1,102 @@
+<script setup lang="ts">
+import { onMounted, ref } from 'vue';
+
+import { Card, Divider } from '@/lib/primevue';
+import { userService } from '@/services';
+import { formatDateShort } from '@/utils/formatters';
+
+import type { Ref } from 'vue';
+import type { Note } from '@/types';
+
+// Props
+type Props = {
+  note: Note;
+  submissionId: string;
+};
+
+const props = withDefaults(defineProps<Props>(), {});
+
+// State
+const userName: Ref<string> = ref('');
+
+onMounted(async () => {
+  if (props.note.createdBy) {
+    const res = (await userService.searchUsers({ identityId: [props.note.createdBy] })).data;
+    userName.value = res.length ? res.pop().fullName : '';
+  }
+});
+</script>
+
+<template>
+  <Card>
+    <template #title>
+      <h3 class="mt-1 mb-1">{{ props.note.title }}</h3>
+      <Divider type="solid" />
+    </template>
+    <template #content>
+      <div class="grid nested-grid">
+        <!-- Left column -->
+        <div class="col-12 md:col-6 lg:col-4">
+          <div class="grid">
+            <p class="col-12">
+              <span class="key font-bold">Date:</span>
+              {{ props.note.createdAt ? formatDateShort(props.note.createdAt ?? '') : undefined }}
+            </p>
+          </div>
+        </div>
+        <!-- Middle column -->
+        <div class="col-12 md:col-6 lg:col-4">
+          <div class="grid">
+            <p class="col-12">
+              <span class="key font-bold">Author:</span>
+              {{ userName }}
+            </p>
+          </div>
+        </div>
+        <!-- Right column -->
+        <div class="col-12 md:col-6 lg:col-4">
+          <div class="grid">
+            <p class="col-12">
+              <span class="key font-bold">Note type:</span>
+              {{ props.note.noteType }}
+            </p>
+          </div>
+        </div>
+        <p class="col-12 mt-0 mb-0 note-content">{{ props.note.note }}</p>
+      </div>
+    </template>
+  </Card>
+</template>
+
+<style scoped lang="scss">
+h2 {
+  margin: 0;
+}
+
+p {
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+.key {
+  color: #38598a;
```

Comment:
Consideration: Down the line if there is a way to reference a set of color names in the global scss that would be better than the magic constants defined here, in the event this specific color hex is used in other places.

Reply:
Looking through the app it does seem to be used in a few places. Navbar, NoteCard, PermitCard, and SubmissionForm components uses that colour. The hexcode is from the gov-wide design reference, so it should definitely go into the global scss. But agreed that this should be done down the line.